packages/EasyCore/src/Bridge/Laravel/Middleware/TrimStrings.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyCore\Bridge\Laravel\Middleware;
+
+use EonX\EasyCore\Helpers\StringsTrimmerInterface;
+use Illuminate\Http\Request;
+use Symfony\Component\HttpFoundation\ParameterBag;
+
+final class TrimStrings
+{
+    /**
+     * A list of array keys whose values will be ignored during processing.
+     * @see \EonX\EasyCore\Tests\Helpers\RecursiveStringsTrimmerTest for example
+     *
+     * @var string[]
+     */
+    private $except;
+
+    /**
+     * @var \EonX\EasyCore\Helpers\StringsTrimmerInterface
+     */
+    private $trimmer;
+
+    /**
+     * @param string[] $except
+     */
+    public function __construct(StringsTrimmerInterface $trimmer, array $except = [])
```

Comment:
Please use nullable to define optional parameters `?array $except = null`.

Reply:
Done.